client/src/locales/en-US.json

```diff
@@ -14,8 +14,9 @@
         "yes": "Exit",
         "no": "Cancel"
     },
-    "openFile": {
-        "failed": "Failed to open entry at `{path}`."
+    "globalErrors": {
+        "openFileFailed": "Failed to open entry at `{path}`.",
+        "incompatibleServer": "Your client and the server are not compatible. Online features will not be available. Please check that your client is up-to-date, and if so, contact an administrator."
```

Comment:
`client` isn't meaningful to the user, I would have replaced it with `application` instead.